_docs/adding-datasets.md

```diff
@@ -108,4 +111,4 @@ $ docker-compose restart
 ## FAQ
 
 * If a dataset fails to load, you can see logs under `/erddap/data/logs`.
-* If a dataset fails to load, visit <http://localhost:8080/erddap/status.html> (which flushes log information to the log file), then look for an error message related to the dataset in `/erddap/data/logs/log.txt`.
\ No newline at end of file
+* If a dataset fails to load, visit <http://localhost:8080/erddap/status.html> (which flushes log information to the log file), then look for an error message related to the dataset in `/erddap/data/logs/log.txt`.
```

Comment:
Not sure why this line was modified.

Reply:
Just white space that was probably removed by your EDIT config. You can omit those changes in GH reviews with by clicking here:

![Screenshot from 2023-01-31 18-13-41](https://user-images.githubusercontent.com/950575/215884650-e4ffd191-a2ef-4215-af19-1e5ee2033ab0.png)